packages/office-ui-fabric-react/src/components/Icon/Icon.tsx

```diff
@@ -54,14 +54,14 @@ export const Icon = (props: IIconProps): JSX.Element => {
           'aria-hidden': true,
           'data-icon-name': iconName,
         }) }
+        onClick={ onClicked }
```

Comment:
it would be nice to rename "onClicked" here.

Reply:
rename to what? Not sure the purpose.

Reply:
`deprecatedDoNotUseOnClicked`? ;)

Reply:
well I can't change the prop, so just alias it somewhere so that line 57 is using something that screams deprecation?

Also to note, this onClicked was added specifically for a calendar update a few weeks ago. I fixed that reference, so it might be worth ripping it out.

Reply:
Was joking with my name ;) yeah.. maybe make the breaking change would be fine here? If for whatever reason someone took a dependency on that it would be very easy to fix